dynamic_values.tf

```diff
@@ -47,8 +47,7 @@ module "unit_tests" {
   }
   security_group_rules = [
     {
-      name  = "test-rule"
-      field = "field"
+      name = "test-rule"
```

Comment:
according to the comment https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/pull/3/files#r898114119 `field` is not needed anymore